### PR TITLE
STSMACOM-331: Increase custom field coverage to 80%

### PR DESCRIPTION
**Pull request by BogdanDenis**
## Purpose
Currently we have a tech debt to cover Custom Fields functionality with tests and increase coverage of code related to this feature. This PR covers writing tests for Custom Fields functionality

### CODE REVIEW

**Comment by zburke** [comment]
> The problem is that I haven't found a way to mock `store.discovery.modules` object during testing, so during testing requests weren't being sent. 

Did you figure this out what you needed here? It isn't really clear to me what you are asking. 

The `discovery.modules` slice of the store is copied onto `props.stripes` so you can access it there. Alternatively, if you need to modify the way `stripes-core` is configured for testing, that happens in the [setup-application](https://github.com/folio-org/stripes-core/blob/master/test/bigtest/helpers/setup-application.js) and [stripes-config](https://github.com/folio-org/stripes-core/blob/master/test/bigtest/helpers/stripes-config.js) test-helpers.

**Comment by zburke** [comment]
Also, I'm not thrilled about turning off the password-validation test; on master, it passes locally and in CI.

File: lib/CustomFields/utils/useCustomFieldsFetch.js [hunk 1]
```diff
@@ -29,9 +29,7 @@ const useCustomFieldsFetch = (okapi, backendModuleId, entityType) => {
       }
     };
 
-    if (backendModuleId) {
```
Comment (MaksymDryha) [inline lib/CustomFields/utils/useCustomFieldsFetch.js:32]: This check shouldn't be removed. It's possible that the backend doesn't manage to send the response to the `/_/proxy/tenants/diku/modules` request before `useCustomFieldsFetch` is called. This mainly applies to situations when a user visits a page, that fetches custom fields, using a direct link (`/settings/users/custom-fields`, `/users/preview/some-id`, etc). In such cases, this change will lead to making a request for custom fields even though we don't know the `backendModuleId` yet.

File: lib/CustomFields/components/CustomFieldsForm/tests/CustomFieldsForm-test.js [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+    expect(customFieldsForm.sectionTitleValue).to.equal('Custom fields');
+  });
+
+  describe('when clicking on an accordion', () => {
+    let initialOpenState;
+
+    beforeEach(async () => {
+      await customFieldsForm.whenLoaded();
+
+      customFieldsForm.customFieldAccordions(0).isOpen;
```
Comment (MaksymDryha) [inline lib/CustomFields/components/CustomFieldsForm/tests/CustomFieldsForm-test.js:86]: is this needed?

Comment (BogdanDenis) [reply lib/CustomFields/components/CustomFieldsForm/tests/CustomFieldsForm-test.js:86]: nope, that's what's left of a `console.log`:)

File: lib/CustomFields/utils/useCustomFieldsFetch.js (hunk 1 above)
Comment (BogdanDenis) [reply lib/CustomFields/utils/useCustomFieldsFetch.js]: Reverted this change in a new commit

**Comment by BogdanDenis** [comment]
@zburke thanks for pointing me in the right direction, I've resolved those problems and updated PR

**Pull request merged**